corehq/apps/export/forms.py

```diff
@@ -344,7 +345,7 @@ def get_multimedia_task_kwargs(self, export, download_id):
             'app_id': export.app_id,
             'xmlns': export.xmlns if hasattr(export, 'xmlns') else '',
             'export_id': export.get_id,
-            'zip_name': 'multimedia-{}'.format(export.name),
+            'zip_name': 'multimedia-{}'.format(unidecode(export.name)),
```

Comment:
hmm, maybe i shouldn't have merged. i think this is fine but would:
```python
'zip_name': u'multimedia-{}'.format(export.name)
```
also not have worked? I think unicode filenames are ok in general, no?